find-missing-translations.js

```diff
@@ -0,0 +1,71 @@
+var {parse} = require('path')
```

Comment:
I think if you add `/* eslint-env node*/` to the top of this file most of the lint failures should be fixed. @dudethatbe